File: 프로그래머스/1/135808. 과일 장수/과일 장수.java
import java.util.*;
import java.util.stream.Collectors;

class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        List<Integer> arr = Arrays.stream(score).boxed().collect(Collectors.toList());
        Collections.sort(arr, Collections.reverseOrder());
        
        for(int i = m - 1; i < arr.size(); i+=m){
            answer += arr.get(i) * m;
        }
        return answer;
    }
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.swift
import Foundation

func solution(_ k:Int, _ m:Int, _ score:[Int]) -> Int {
    var result = 0
    var count = 0
    var scoreLen = score.count
    var sortArr = score
    sortArr.sort(by: >)
    while(count < scoreLen) {
        if count + m - 1 >= scoreLen {
            break
        } else {
            var num = sortArr[count + m - 1]
            result += num * m
        }
        count += m
    }
    return result
}